<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerPage</class>
 <widget class="QWidget" name="registerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4486c3;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="loginBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>531</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #5f9cd5;
border-radius: 70px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="registerTitle">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 40pt &quot;Allerta&quot;;
border:0px;
color:white;
font-weight:bold;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string>REGISTER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUsername">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt &quot;Allerta&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#9bc0e2;
border-radius:15px;
border:0px;
color:white;
font:20px &quot;Allerta&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#9bc0e2;
border-radius:15px;
border:0px;
color:white;
font:20px &quot;Allerta&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#9bc0e2;
border-radius:15px;
border:0px;
color:white;
font:20px &quot;Allerta&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>490</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#9bc0e2;
border-radius:25px;
border:0px;
font:15pt &quot;Allerta&quot;;
color:white;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Register now</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>270</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt &quot;Allerta&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelConfirmPassword">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>360</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt &quot;Allerta&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Confirm Password&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelText">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>560</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt &quot;Allerta&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Already have an account?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>600</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:0px;
font:17pt &quot;Allerta&quot;;
color:#9bc0e2;
</string>
   </property>
   <property name="text">
    <string>Log In</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoTitle">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>405</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Crimson Text Semibold</family>
     <pointsize>42</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">letter-spacing:5px;
font: 600 42pt &quot;Crimson Text Semibold&quot;;
border:0px;
color:white;
font-weight:bold;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Panel</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UNIVERSE&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="logoAG">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>80</y>
     <width>461</width>
     <height>421</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../assets/images/Test_logo_2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>460</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4486c3;</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #4486c3;</string>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>A</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>V</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>G</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
